citesphere/src/main/java/edu/asu/diging/citesphere/core/zotero/IZoteroConnector.java

```diff
@@ -67,5 +67,9 @@ void clearCollectionItemsCache(IUser user, String groupId, String collectionId,
     ZoteroResponse<Collection> getCitationCollectionVersions(IUser user, String groupId, Long lastGroupVersion);
 
     ZoteroResponse<Collection> getCitationCollectionsByKey(IUser user, String groupId, List<String> keys);
+    
+    void deleteItem(IUser user, String groupId, String citationKey, Long citationVersion) throws ZoteroConnectionException, ZoteroHttpStatusException;
+    
+    void deleteMultipleCitations(IUser user, String groupId, String citationKey, Long citationVersion) throws ZoteroConnectionException, ZoteroHttpStatusException;
```

Comment:
since all the other methods use `Item` instead of `Citation` this one should be too for consistency.